util/Order.cpp

```diff
@@ -293,28 +293,12 @@ bool FleetMoveOrder::Check(int empire_id, int fleet_id, int dest_system_id, bool
         return false;
     }
 
-    int start_system = fleet->SystemID();
-    if (start_system == INVALID_OBJECT_ID)
-        start_system = fleet->NextSystemID();
-
     auto dest_system = GetEmpireKnownSystem(dest_system_id, empire_id);
     if (!dest_system) {
         ErrorLogger() << "Empire with id " << empire_id << " ordered fleet to move to system with id " << dest_system_id << " but no such system is known to that empire";
         return false;
     }
 
-    // verify fleet route first system
-    if (append && !fleet->TravelRoute().empty()) {
```

Comment:
I think this was the only use of `append` so it could be removed from the function signature...?

Reply:
The AIWrapper currently expects same parameters for the constructor and ```::Check```
https://github.com/freeorion/freeorion/blob/4a704df4d0fd2510b1d6ffbe7da502604cedaabb/python/AI/AIWrapper.cpp#L95-L105